app/backend/aquifers/views.py

```diff
@@ -294,11 +302,47 @@ def delete(self, request, aquifer_id):
         return HttpResponse(status=204)
 
 
-class AquifersSpatial(APIView):
-
-    permission_classes = (AllowAny,)
-
-    def get(self, request):
+AQUIFER_PROPERTIES = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    title='GeoJSON Feature properties.',
+    description='See: https://tools.ietf.org/html/rfc7946#section-3.2',
+    properties={
+        'aquifer_id': get_model_feature_schema(Aquifer, 'aquifer_id'),
+        'name': get_model_feature_schema(Aquifer, 'aquifer_name'),
+        'location': get_model_feature_schema(Aquifer, 'location_description'),
+        'material': get_model_feature_schema(AquiferMaterial, 'description'),
+        'subtype': get_model_feature_schema(AquiferSubtype, 'description'),
+        'vulnerability': get_model_feature_schema(AquiferVulnerabilityCode, 'description'),
+        'productivity': get_model_feature_schema(AquiferProductivity, 'description'),
+        'demand': get_model_feature_schema(AquiferDemand, 'description'),
+        'water_use': get_model_feature_schema(WaterUse, 'description'),
+        'quality_concern': get_model_feature_schema(QualityConcern, 'description'),
+        'litho_stratographic_unit': get_model_feature_schema(Aquifer, 'litho_stratographic_unit'),
+        'mapping_year': get_model_feature_schema(Aquifer, 'mapping_year'),
+        'notes': get_model_feature_schema(Aquifer, 'notes')
+    })
+
+
+@swagger_auto_schema(
+    operation_description=('Get GeoJSON (see https://tools.ietf.org/html/rfc7946) dump of aquifers.'),
+    method='get',
+    manual_parameters=[GEO_JSON_PARAMS],
+    responses={
+        302: openapi.Response(GEO_JSON_302_MESSAGE),
+        200: openapi.Response(
+            'GeoJSON data for aquifers.',
+            get_geojson_schema(AQUIFER_PROPERTIES, 'Polygon'))
+        })
+@api_view(['GET'])
+def aquifer_geojson(request):
+    realtime = request.GET.get('realtime') in ('True', 'true')
+    if realtime:
+        iterator = GeoJSONIterator(AQUIFERS_SQL, AQUIFER_CHUNK_SIZE, connection.cursor())
+        response = StreamingHttpResponse((item for item in iterator),
+                                         content_type='application/json')
+        response['Content-Disposition'] = 'attachment; filename="aquifers.json"'
+        return response
```

Comment:
was not able to use this on https://gwells-dev-pr-1245.pathfinder.gov.bc.ca/gwells/api/v1/gis/aquifers?realtime=true